### PR TITLE
fix: display Uniswap token list in UI

### LINKED ISSUE #2756: Display Uniswap token list in UI
the Uniswap Labs default token list should be a visible option in the token list menu

* logo: brand
* name: Uniswap Labs Token List
* state: on by default
* Gemini token list should continue to be on by default as well

### PR DESCRIPTION
**Pull request by bkrochta**
Fixes #2756. 
Removes Uniswap default-token-list build dependency in favor of treating Uniswap token-list like every other list.

### CODE REVIEW

File: src/constants/lists.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const UNI_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.org'
```
Comment (moodysalem) [inline src/constants/lists.ts:1]: we should use the ENS name `tokens.uniswap.eth`, which points to `ipns://tokens.uniswap.org`, which points at the latest version of the Uniswap token list.

File: src/state/lists/hooks.ts [hunk 2]
```diff
@@ -116,7 +113,7 @@ export function useInactiveListUrls(): string[] {
 export function useCombinedActiveList(): TokenAddressMap {
   const activeListUrls = useActiveListUrls()
   const activeTokens = useCombinedTokenMapFromUrls(activeListUrls)
-  return useMemo(() => combineMaps(activeTokens, TRANSFORMED_DEFAULT_TOKEN_LIST), [activeTokens])
+  return activeTokens
```
Comment (moodysalem) [inline src/state/lists/hooks.ts:116]: this means that users can disable all lists and see no tokens now @willhennessy

**Review by moodysalem — CHANGES_REQUESTED** [review]

**Comment by moodysalem** [comment]
Thank you for the PR @bkrochta, can you also merge main into this branch? To get the linter checks to run

File: src/constants/lists.ts (hunk 1 above)
Comment (bkrochta) [reply src/constants/lists.ts]: Fixed

**Comment by willhennessy** [comment]
Thank bkrochta! Two comments
1. The 'view list' URL is not working. https://tokenlists.org/token-list?url=tokens.uniswap.eth is failing to find a result. Should we try https://tokenlists.org/token-list?url=https://gateway.ipfs.io/ipns/tokens.uniswap.org instead?
2. We should rename the list to "Uniswap Labs Default List". We'll need to make this change in the uniswap token list repo and deploy a new version of the list

**Comment by bkrochta** [comment]
It seems `tokens.uniswap.eth` is not getting resolved correctly by some services. 

`uniswap.tokens.eth` points to `ipns://tokens.uniswap.org`, but this is a DNSLink name which is being deprecated by ens (see [here](https://github.com/ensdomains/ens-app/issues/849#issuecomment-776819864) and [here](https://github.com/ensdomains/content-hash/pull/5)). 

However, the Uniswap interface can still resolve `tokens.uniswap.eth` because it uses its own contenthash decoder [here](https://github.com/Uniswap/interface/blob/a5b152da065a992911d8d8ed7bacb9fd61d1120b/src/utils/contenthashToUri.ts#L21). Other sites such as tokenlists.org use other services to resolve which may not be able to decode a DNSLink contenthash. 

Other tokenlists that use an ens name such as `tokenlist.aave.eth` resolve to `ipfs://{cid}`, so they have no issue being resolved by the Uniswap interface or tokenslist.org or any other service.

This can be fixed by changing `tokens.uniswap.eth` back to `https://gateway.ipfs.io/ipns/tokens.uniswap.org` or changing the contenthash of `tokens.uniswap.eth`

**Comment by willhennessy** [comment]
Thanks for the thorough answer! @moodysalem do you have an opinion on the two options presented in the last sentence? I'm in favor of any minimal effort change to make the 'View list' link in the app work. 

Also, we've updated the name of the token list to `Uniswap Labs List` so that is ready to go.

**Comment by moodysalem** [comment]
does `ipns://tokens.uniswap.org` work?

**Comment by moodysalem** [comment]
> The list will load for the Uniswap interface, but the 'view list' URL https://tokenlists.org/token-list?url=ipns://tokens.uniswap.org will fail since the tokenlist website only expects URLs that start with "https://" or end with ".eth"

Unfortunate, let's just use https://tokens.uniswap.org then.

**Review by moodysalem — APPROVED** [review]

**Pull request merged**